shared/src/components/Header/Header.tsx

```diff
@@ -336,17 +352,27 @@ export const Header = ({
                       </g>
                     </svg>
                   ),
-                  url: authUser?.is_superuser ? "/dashboard" : "/machines",
+                  url: homepageLink.url,
+                },
+                {
+                  className: "p-navigation__link",
+                  "aria-label": homepageLink.label,
+                  "aria-current": isSelected(path, homepageLink, appendNewBase)
+                    ? "page"
+                    : undefined,
                 },
-                "p-navigation__link",
                 appendNewBase
               )}
             </div>
-            {/* eslint-disable-next-line */}
+            {/* TODO: replace anchor with button */}
+            {/* eslint-disable-next-line jsx-a11y/anchor-is-valid, jsx-a11y/role-supports-aria-props */}
```

Comment:
Would you mind adding an issue for this if it doesn't already exist?

Reply:
Good point! Added in https://github.com/canonical-web-and-design/maas-ui/issues/3454